public/legacy-custom-forms/javascripts/data-entry/form.js

```diff
@@ -969,9 +951,9 @@ function insertDataValues( json )
             {
                 var $field = $( fieldId );
 
-                $field.find( 'input[class="entryfileresource-input"]' ).val( value.val );
+                $field.find( 'input[class="entryfileresource-input"]' ).val( valueToShow );
 
-                var split = dhis2.de.splitFieldId( value.id );
+                var split = dhis2.de.splitFieldId( value.value );
 
```

Comment:
Didn't talk about this in the PR session, but I'm guessing this is a fix for file inputs?